server.js

```diff
@@ -0,0 +1,6 @@
+const http = require('http');
+const app = require('./lib/app.js');
+
+http.createServer(app).listen(4000);
```

Comment:
```suggestion
const PORT = 4000;

http
  .createServer(app)
  .listen(PORT, () => console.log('JAMMIN ON', PORT));
```
Here is another way you can implement your listener function in the server :)